src/sentry/api/endpoints/issues/related_issues.py

```diff
@@ -33,6 +40,8 @@ def get(self, request: Request, group: Group) -> Response:
         :pparam Request request: the request object
         :pparam Group group: the group object
         """
+        serializer = RequestSerializer(data=request.data)
+        serializer.is_valid()
```

Comment:
I think you need to use this to produce an error somehow? and then retrieve the data from this and pass it along